core/sail/spin/src/test/resources/functions/apf/concat.sparql

```diff
@@ -0,0 +1,5 @@
+prefix apf: <http://jena.hpl.hp.com/ARQ/property#>
```

Comment:
You can also use `QueryResultIO.parseTuple()`, which is a static method giving you a `TupleQueryResult` object. You can then use `QueryResults.equals` or `QueryResults.isSubset` to compare your expected and actual results.